src/main/webapp/src/components/CourseDetails.vue

```diff
@@ -206,20 +253,117 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.label {
-    width: 7rem;
-    font-weight: 700 !important;
-    float: left !important;
+// page container
+.course-details-container {
+    margin: auto;
+    max-width: 1200px;
+}
+.course-details-main-container {
+    display: flex;
+    justify-content: space-between;
+}
+// buttons container
+.course-details-nav {
+    margin-bottom: $xxl-space;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav-item > button:first-child {
+    margin-right: $s-space;
+}
+.button-icon {
```

Comment:
sollte das hier nicht in in die globale scss und height explizit setzen?

Reply:
stimmt, sollte auf globale scss :)